cdhweb/pages/templates/cdhpages/blocks/block_quote.html

```diff
@@ -1,3 +1,3 @@
 {% load wagtailcore_tags %}
 
-<div class="block-quote">{{ self.quote|richtext }}</div>
\ No newline at end of file
+<div class="block-quote rich-text">{{ self.quote|richtext }}</div>
```

Comment:
Lack of these rich-text classes was creating extra spacing between the streamfield blocks.